### PR TITLE
lots of various updates to the website

### PR DESCRIPTION
**Pull request by flybayer**
- Messaging updates for 2.0
- Fixes
- Improvements

### CODE REVIEW

File: app/pages/index.js [hunk 1]
```diff
@@ -312,7 +304,7 @@ const Home = ({randomContributors}) => {
                   className="hidden xl:block xl:-top-40 xl:-right-52"
                 />
                 <FeatureIconTitle icon="thumbsUp" title="Authentication & Authorization">
-                  Built-in authentication is super easy and very secure. Works with any identity
+                  Blitz authentication is super easy and very secure. Works with any identity
```
Comment (beerose) [inline app/pages/index.js:307]: Can we rephrase it to avoid `very secure`? Because with `very` it feels like there's a range — and that it's _somewhat_ secure but not like fully secure 😅 
Maybe it's just me, though.

File: app/pages/index.js [hunk 2]
```diff
@@ -200,7 +190,7 @@ const Home = ({randomContributors}) => {
                 <a
                   target="_blank"
                   rel="noopener noreferrer"
-                  href="https://codesandbox.io/s/blitzjs-2-0-codesandbox-demo-b5v8xy?file=/app/blitz-server.ts"
+                  href="https://codesandbox.io/s/flamboyant-meninsky-j63yq?file=/app/projects/mutations/createProject.ts"
```
Comment (beerose) [inline app/pages/index.js:193]: Looks like you reverted the code sandbox change. I added a link to a toolkit version few days ago.

File: app/pages/index.js (hunk 1 above)
Comment (flybayer) [reply app/pages/index.js]: fixed

File: app/pages/index.js (hunk 2 above)
Comment (flybayer) [reply app/pages/index.js]: fixed

**Review by beerose — APPROVED** [review]

**Review by dillonraphael — APPROVED** [review]
